nats/src/main.rs

```diff
@@ -154,6 +157,9 @@ impl ConnectionConfig {
         if config.cluster_uris.is_empty() {
             config.cluster_uris.push(DEFAULT_NATS_URI.to_string());
         }
+        if let Some(tls_ca) = values.get("tls_ca") {
```

Comment:
Nit: can we give this a name and store it next to [the others](https://github.com/wasmCloud/capability-providers/blob/main/nats/src/main.rs#L26)?